### ISSUE TITLE
Accessibility Tests performed

### ISSUE THREAD

**Issue by mouxdesign** [ACINQ/phoenix#290]
Myself and [Christoph](https://github.com/GBKS) did some accessibility testing today on the Phoenix wallet. 

Overall the accessibility was quite good. We found a few things that can be optimized:

- Some labelling issues
- The Qr code recognition 
- The camera option opening and users being informed of this
- Receive screen; Voice over confirmation of the amount and date of sats received<br/>
![Group 1 (1)](https://user-images.githubusercontent.com/90313394/176665928-86592d17-5903-4e6d-970b-9838b0de353c.jpg)

To provide a brief overview of what we are doing:

- Accessibility testing [document](https://docs.google.com/document/d/1zIuLvUPgGfKVIo_WBBteGhzTEUyn9fbbvtUUpEa7QSw/edit#)

- [Figma file](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=106%3A3453) with the entire test we performed

- [Accessibility](https://bitcoin.design/guide/designing-products/accessibility/) article in the bitcoin design guide 

- How we briefly [prepare](https://www.youtube.com/watch?v=n9FvI5SfpoU&t=4s) our phones to perform the tests

**Comment by mouxdesign** [comment]
Have done some follow up tests on the receive flow which I recorded as well. This was a voice over only test performed on the Receive flow of Phoenix. 

**Summary of findings:**
Overall the app is quite accessible with regards to voice over features,  some areas could could be improved are:
* Time is read out as numbers and not as an actual time
* Copy button is labelled as a Tab button and so the screen reader reads it out as “Tabs”
* Pop up modal (Receive screen) is not read out by voice over
* The qr code is not acknowledged by the Voice over
* Desc is read out as desc, however it is the description of the transaction

Link to [google doc](https://docs.google.com/document/d/1bxhZmp0OE_7SvCwKjiyE6ZMKYm1S3PIhN206SueV68k/edit?usp=sharing) with video and notes how the app was analyzed and what actions were performed.

**Comment by dpad85** [comment]
Hello, thanks for testing the app and providing such a detailed feedback. Regarding Android we are completely reworking the app so some of the accessibility issues you found may just disappear, and new ones will certainly be created! Accessibility is something we will look at once the new Android app is ready to ship. Hopefully, jetpack-compose should make things easy to fix.

**Comment by robbiehanson** [comment]
@mouxdesign Thank you so much ! This is incredibly helpful !

**Comment by GBKS** [comment]
[Accessibility day](https://github.com/BitcoinDesign/Meta/issues/492) is coming up on May 18 and in the Bitcoin Design Community we're looking to rally designers and developers to spend that day learning about accessibility and making improvements. Would you be interested in joining? Doesn't have to be a huge effort, could be simply about making some of the easier fixes of what is posted in this issue.

**Comment by robbiehanson** [comment]
Just wanted to note that I spent a fair amount of time fixing a lot of accessibility issues earlier this year. So the iOS tests may need to be updated.

**Comment by robbiehanson** [comment]
> Would it help if we re-tested the application?

Yes, that would be super helpful. We fixed many issues, but then we made large changes across the app for version 2. So I may have broken several things since then.

**Comment by GBKS** [comment]
> We fixed many issues

So awesome to hear. I just did a quick test and the app is in really good shape (Version 2.1.0 on iPhone 15). Thank you for digging into this.

I found a handful of smaller issues that I documented here.

![Phoenix accessibility test summary 240105](https://github.com/ACINQ/phoenix/assets/695901/2c4a9632-7ca7-4fff-b3ec-ba6bfb982e86)

Great job adding in the tap-and-hold actions on the receive screen. A bit hard to discover, but super useful.

I am curious as to what user feedback you are getting for the "add liquidity" flow. I see you're using metaphors to explain the concept, I wonder if it clicks for people. Do you also explain these concepts on onboarding or is the "add liquidity" screen the first time people encounter them?

**Comment by GBKS** [comment]
I recommend closing this issue as it has not seen activity in over 2 years. The test results are probably no longer valid. And with accessibility linters and AI, it should be part of normal coding practices to catch most issues these days.
